src/constants/index.ts

```diff
@@ -0,0 +1,209 @@
+		icon: 'FileText',
+		searchString: 'attachment:word'
+	},
+	{
+		label: t('attachment_type.zipped_file', 'Zipped File'),
+		icon: 'FileZip',
+		searchString: 'attachment:application/zip OR attachment:application/x-zip-compressed'
+	}
+];
+
+type EmailStatusItemsConstantProps = {
+	label: string;
+	searchString: string;
+};
+
+export const emailStatusItemsConstant = (t: TFunction): Array<EmailStatusItemsConstantProps> => [
+	{
+		label: t('email_status.read', 'read'),
+		searchString: 'is:read'
+	},
+	{
+		label: t('email_status.unread', 'unread'),
+		searchString: 'is:unread'
+	},
+	{
+		label: t('email_status.flagged', 'flagged'),
+		searchString: 'is:flagged'
+	},
+	{
+		label: t('email_status.unflagged', 'not flagged'),
+		searchString: 'is:unflagged'
+	},
+	{
+		label: t('email_status.draft', 'draft'),
+		searchString: 'is:draft'
+	},
+	{
+		label: t('email_status.sent', 'sent by me'),
+		searchString: 'is:sent'
+	},
+	{
+		label: t('email_status.received', 'received by me'),
+		searchString: 'is:recevided'
+	},
+	{
+		label: t('email_status.replied', 'answered by me'),
+		searchString: 'is:replied'
+	},
+	{
+		label: t('email_status.unreplied', 'not answered by me'),
+		searchString: 'is:unreplied'
+	},
+	{
+		label: t('email_status.forwarded', 'forwarded'),
+		searchString: 'is:forwarded'
+	},
+	{
+		label: t('email_status.unforwarded', 'not forwarded'),
+		searchString: 'is:unforwarded'
+	},
+	{
+		label: t('email_status.invitations', 'invitations'),
+		searchString: 'is:invite'
+	},
+	{
+		label: t('email_status.solo', 'conversations with a single message'),
+		searchString: 'is:solo'
+	},
+	{
+		label: t('email_status.fromme', 'from me'),
+		searchString: 'is:fromme'
+	},
+	{
+		label: t('email_status.tome', 'to me'),
+		searchString: 'is:tome'
+	},
+	{
+		label: t('email_status.ccme', 'in copy to me'),
+		searchString: 'is:ccme'
+	},
+	{
+		label: t('email_status.tofromme', 'to me, from me or in copy to me'),
+		searchString: 'is:tofromme'
+	},
+	{
+		label: t('email_status.toccme', 'to me or in copy to me'),
+		searchString: 'is:toccme'
+	},
+	{
+		label: t('email_status.fromccme', 'from me'),
+		searchString: 'is:fromccme'
+	},
+	{
+		label: t('email_status.tofromccme', 'to me, from me or in copy to me'),
+		searchString: 'is:tofromccme'
+	},
+	{
+		label: t('email_status.local', 'on my folders'),
+		searchString: 'is:local'
+	},
+	{
+		label: t('email_status.remote', 'on a folder shared by me'),
+		searchString: 'is:remote'
+	},
+	{
+		label: t('email_status.anywhere', 'in all the folders'),
+		searchString: 'is:anywhere'
+	}
+];
+
+export const MAIL_APP_ID = 'carbonio-mails-ui';
+
+export const FOLDER_VIEW = {
+	search_folder: 'search folder',
+	tag: 'tag',
+	conversation: 'conversation',
+	message: 'message',
+	contact: 'contact',
+	document: 'document',
+	appointment: 'appointment',
+	virtual_conversation: 'virtual conversation',
+	remote_folder: 'remote folder',
+	wiki: 'wiki',
+	task: 'task',
+	chat: 'chat'
```

Comment:
An Enum can also be used here instead of object and the Key should be in Capital letters

Reply:
you are right, I will make a note to create a ticket for it